bundles/org.openhab.core/src/main/java/org/openhab/core/library/items/RollershutterItem.java

```diff
@@ -80,4 +81,13 @@ public void setState(State state) {
             logSetTypeError(state);
         }
     }
+
+    @Override
+    public void setTimeSeries(TimeSeries timeSeries) {
+        if (timeSeries.getStates().allMatch(s -> s.state() instanceof PercentType)) {
```

Comment:
Note that `UpDownType` are also valid states for a Rollershutter items. And I could imagine that UP, DOWN are possibly more often used here than 0 and 100.